src/pages/MyNotes/types.ts

```diff
@@ -0,0 +1,29 @@
+import { Dispatch } from 'react';
+
+import { INote } from 'types';
+
+interface INotesList {
+  notes?: INote[][];
+  activeNote: INote | null;
```

Comment:
could it be replaced with undefined or it should be null?

```suggestion
  activeNote?: INote;
```